src/OpenTK.NT/Native/User32/Structs/RawInputData.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Runtime.InteropServices;
+
+namespace OpenTK.NT.Native
+{
+    [StructLayout(LayoutKind.Explicit)]
+    public struct RawInputData
+    {
+        [FieldOffset(0)] public RawMouse Mouse;
+        [FieldOffset(0)] public RawKeyboard Keyboard;
+        [FieldOffset(0)] public RawHid Hid;
+
+        public static readonly uint SizeInBytes = (uint)Marshal.SizeOf<RawInputData>();
```

Comment:
Is this required?

Reply:
It's used in `WinGLNative` for a `GetMouseMovePointEx` call, since that function requires the struct size as the first parameter.